<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ryerson University Burst</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Accounts</string>
    </property>
    <widget class="QPushButton" name="pushButton_AccountsBrowser">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browser</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Accounts">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="listView_Accounts">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Proxies">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>351</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Proxies</string>
    </property>
    <widget class="QPushButton" name="pushButton_Proxies_Browser">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browser</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Proxies">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="listView_Proxies">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>311</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_PerProxyRequestLimit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Proxy Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Per Proxy Request Limit:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_ProxyTypeHttp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Http</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_ProxyTypeSocks5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Socks5</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="pushButton_Pause">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Start">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Multi Threads:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeout(Sec):</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_MultiThreads">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Timeout">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sqlite3 DB:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_db">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval(Sec):</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Interval">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Arguments">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>741</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Request Arguments</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Url:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Url">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>161</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Argv[1]</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv1Key">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv1Value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>161</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Argv[2]</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv2Key">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv2Value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>161</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Argv[3]</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv3Key">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv3Value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>161</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Argv[5]</string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv5Key">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv5Value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>161</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Argv[6]</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv6Key">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv6Value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>150</y>
       <width>161</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Argv[7]</string>
     </property>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv7Key">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv7Value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>150</y>
       <width>161</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Argv[8]</string>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv8Key">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv8Value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>161</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Argv[4]</string>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Key:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv4Key">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Argv4Value">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Accounts Line:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>550</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CurrentAccountsLine">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_UseProxie">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>520</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxy_Enable</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Profile"/>
    <addaction name="actionSave_Profile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Profile">
   <property name="text">
    <string>Load Profile</string>
   </property>
  </action>
  <action name="actionSave_Profile">
   <property name="text">
    <string>Save Profile</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
